tools/thinpool.sh

```diff
@@ -13,9 +13,9 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-set -eu
+FICD_DM_VOLUME_GROUP="$FICD_DM_VOLUME_GROUP"
```

Comment:
Is this assignment intentional ?

Reply:
Yes. If FICD_DM_VOLUME_GROUP is not defined, I'd like to treat that as an empty string.